_pkgdown.yml

```diff
@@ -1,4 +1,6 @@
 url: https://epiforecasts.io/socialmixr/
+
 template:
   bootstrap: 5
-
+  bslib:
+    font_weight_base : 300
```

Comment:
I would recommend against decreasing the font weight below the normal value (400) as this impacts accessibility.

https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight

https://accessibility.psu.edu/2021/10/beware-light-weight-fonts/

Reply:
The first ref suggests to avoid 100 and 200, whereas the secone one gives no specific recommendation.

That said, probably best to err on the side of caution in this context - will revert.